src/main/frontend/Admin.html

```diff
@@ -197,88 +238,101 @@ <h1>Flights Database</h1>
-
-    // Create a form element
-    let form = document.createElement("form");
-    form.className = "editform"
-    let fields
-
-    // Define the fields you want to edit
-    if (row.cells[6].innerHTML === "EHAM")
-        fields = ["Callsign", "Operator", "ICAOType", "ADEP", "DEST", "TAS", "RFL", "WeightClass", "RunWay", "GATE", "STACK"];
-    else
-        fields = ["Callsign", "Operator", "ICAOType", "ADEP", "DEST", "TAS", "RFL", "WeightClass"];
-
-    // Iterate over the fields and create textboxes for each
-    fields.forEach(field => {
-        let label = document.createElement("label");
-        label.textContent = `Enter the new ${field}: `;
-        let input = document.createElement("input");
-        input.type = "text";
-        input.value = row.cells[field === "RunWay" ? 11 : fields.indexOf(field) + 2].innerHTML; // Adjust the cell index for RunWay
-        input.name = field;
-        form.appendChild(label);
-        form.appendChild(input);
-        form.appendChild(document.createElement("br"));
-    });
-
-    // Add a submit button to the form
-    let submitButton = document.createElement("button");
-    submitButton.textContent = "Submit";
-    form.appendChild(submitButton);
-
-    // Create a modal overlay
-    let modalOverlay = document.createElement("div");
-    modalOverlay.classList.add("modal-overlay");
-
-    // Display the form in the modal overlay
-    modalOverlay.appendChild(form);
-    document.body.appendChild(modalOverlay);
-
-    // Handle the form submission
-    form.addEventListener("submit", function (event) {
-        event.preventDefault();
-
-        // Extract values from the form
-        let updatedData = { FLIGHT_ID: row.cells[1].innerHTML };
+        let row = button.parentNode.parentNode;
+
+        // Create a form element
+        let form = document.createElement("form");
+        form.className = "editform"
+        let fields
+
+        // Define the fields you want to edit
+        if (row.cells[6].innerHTML === "EHAM")
+            fields = ["Callsign", "Operator", "ICAOType", "ADEP", "DEST", "FLIGHT_RULES", "TAS", "RFL", "WeightClass", "RunWay", "GATE", "STACK"];
+        else
+            fields = ["Callsign", "Operator", "ICAOType", "ADEP", "DEST", "FLIGHT_RULES", "TAS", "RFL", "WeightClass"];
+
+        // Iterate over the fields and create textboxes for each
         fields.forEach(field => {
-            updatedData[field] = form.elements[field].value;
-        });
-        console.log(JSON.stringify(updatedData))
-        // Perform the fetch request
-        fetch('http://127.0.0.1:5000/scene/update', {
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json',
-            },
-            body: JSON.stringify(updatedData),
-        })
-        .then(response => {
-            if (!response.ok) {
-                throw new Error(`HTTP error! Status: ${response.status}`);
+            let label = document.createElement("label");
```

Comment:
You can just resolve this afther response.

Are you sure you want to to this via hardcoding the values. Even though I dont know the exact reason why it happends I do beleve that it could be fixed without a work around what you have right now?